app/src/main/java/com/sdp13epfl2021/projmag/activities/ProjectInformationActivity.kt

```diff
@@ -133,6 +141,47 @@ class ProjectInformationActivity : AppCompatActivity() {
         }
     }
 
+    private fun addToFavoritesButton(favButton: Button){
+        val userDataDatabase = Utils.getInstance(this).userDataDatabase
```

Comment:
As it is use many times (here and 119), this should be maybe be saved somewhere.